### PR TITLE
Adding element to HTML function

### PR DESCRIPTION
**Pull request by rafikkasmi**

### CODE REVIEW

File: soup.go [hunk 1]
```diff
@@ -293,6 +293,17 @@ checkNode:
...
 }
 
+// HTML returns the HTML code for the specific element
+
+func (r Root) HTML() string {
+	var buf bytes.Buffer
+	e := html.Render(&buf, r.Pointer)
+	if e != nil {
+		return ""
+	}
```
Comment (cbednarski) [inline soup.go]: In Go these preceding lines are normally written using the following idiom:
```
if err := html.Render(&bug, r.Pointer); err != nil {
	return ""
}
```
This accomplishes two things:

1. It makes it clear that the variable (`err`) is an error and that we are handling the error right away.
2. The `err` variable is scoped to the `if` block so it does not leak out into other logic that follows.

Comment (anaskhan96) [reply soup.go]: +1

**Comment by anaskhan96** [comment]
PR looks good @rafikkasmi 
Can you go ahead and write a test + documentation in the README for the `HTML()` method? That would make it complete with verified testing.

**Comment by anaskhan96** [comment]
Merging this, I would be making changes for test + documentation for this myself.

**Pull request merged**